### PR TITLE
RULEAPI-680: Remove the mention of [label]

### PR DESCRIPTION
**Pull request by marco-antognini-sonarsource**
Update documentation and script.

---

FWIW, the doc already contains (~L260):

> == Untriaged Pull Requests
Untriaged PRs are the ones without any label. You can easily see all of them with the filter Label > Unlabeled.
All triaged PRs should have at least one label that corresponds to the bubble(s) the PR is related to. This allows bubbles to easily filter out the PRs they are interested in.

So the requirement to have labels is already there. Let me know if this should be reworded to have more visibility.

---

I also didn't document how to use `git` commands to extract the history for a given language. _If_ it should be included, I'm not exactly sure where this should go. It also feels out of scope somehow, but I don't have a strong opinion on this point.

### CODE REVIEW

**Comment by arseniy-sonar** [comment]
>FWIW, the doc already contains (~L260):
> So the requirement to have labels is already there. Let me know if this should be reworded to have more visibility.

The documentation mentions the GitHub PR labels (not the `[label]` markers in PR titles/commit messages). Perhaps it is worth rephrasing with more precise language.

**Review by arseniy-sonar — APPROVED** [review]
Good!
See my comment above if you want to clarify the language to make sure people understand it talks about the GitHub PR labels.
I don't think it is necessary to document the git commands to classify commits by language, as I don't know of any use case for that at the moment.

File: README.adoc [hunk 1]
```diff
@@ -258,8 +258,11 @@ This way, your comment will only be visible in GitHub preview and on the Search
 
 == Untriaged Pull Requests
 
-Untriaged PRs are the ones without any label. You can easily see all of them with the filter https://github.com/SonarSource/rspec/pulls?q=is%3Aopen+is%3Apr+no%3Alabel[`Label > Unlabeled`]. +
```
Comment (marco-antognini-sonarsource) [inline README.adoc:260]: @arseniy-sonar I tried to rephrase it just a little to make it less ambiguous for people used to the old ways (i.e. `[lang]` in the PR's title).

The text should also be unambiguous for newcomers who don't know about `[lang]` by not mentioning this form at all.

Did I succeed? :-)

Comment (arseniy-sonar) [reply README.adoc:260]: Yep, looking nice!

**Pull request merged**